contracts/zaps/steth/ZapSteth.sol

```diff
@@ -0,0 +1,190 @@
+// SPDX-License-Identifier: AGPL-3.0
+pragma solidity 0.6.12;
+pragma experimental ABIEncoderV2;
+
+import "../../../interfaces/curve/Curve.sol";
+import "../../../interfaces/lido/ISteth.sol";
+
+import "@openzeppelin/contracts/math/Math.sol";
+import "@openzeppelin/contracts/access/Ownable.sol";
+
+import {
+    SafeERC20,
+    SafeMath,
+    IERC20,
+    Address
+} from "@openzeppelin/contracts/token/ERC20/SafeERC20.sol";
+import "../BaseZap.sol";
+
+contract ZapSteth is BaseZap {
+    using SafeERC20 for IERC20;
+    using Address for address;
+    using SafeMath for uint256;
+
+    ISteth public stETH =
+        ISteth(address(0xae7ab96520DE3A18E5e111B5EaAb095312D7fE84));
+
+    //slippage allowance is out of 10000. 20 is 2%
+    ICurveFi public StableSwapSTETH =
+        ICurveFi(address(0xDC24316b9AE028F1497c275EB9192a3Ea0f67022));
+
+    uint256 public constant DEFAULT_SLIPPAGE = 50;
```

Comment:
make this a contract field thats mutable and add setters for governable, if slippage increases for some reason above this we may need to change settings